src/renderer/views/pool/PoolHistoryView.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback, useRef } from 'react'
+
+import * as RD from '@devexperts/remote-data-ts'
+import { Asset, assetToString } from '@xchainjs/xchain-util'
+import * as FP from 'fp-ts/function'
+import * as O from 'fp-ts/Option'
+import { useObservableState } from 'observable-hooks'
+import * as RxOp from 'rxjs/operators'
+
+import { PoolActionsHistory } from '../../components/poolActionsHistory'
+import { Filter } from '../../components/poolActionsHistory/types'
+import { useMidgardContext } from '../../contexts/MidgardContext'
+import { useThorchainContext } from '../../contexts/ThorchainContext'
+import { liveData } from '../../helpers/rx/liveData'
+import { DEFAULT_ACTIONS_HISTORY_REQUEST_PARAMS, LoadActionsParams } from '../../services/midgard/poolActionsHistory'
+import { PoolActionsHistoryPage, PoolActionsHistoryPageRD } from '../../services/midgard/types'
+
+type Props = {
+  poolAsset: Asset
+  className?: string
+}
+
+const DEFAULT_REQUEST_PARAMS = {
+  ...DEFAULT_ACTIONS_HISTORY_REQUEST_PARAMS,
+  itemsPerPage: 5
+}
+
+export const PoolHistory: React.FC<Props> = ({ className, poolAsset }) => {
+  const {
+    service: { poolActionsHistory }
+  } = useMidgardContext()
+
+  const { getExplorerTxUrl$ } = useThorchainContext()
+
+  const prevActionsPage = useRef<O.Option<PoolActionsHistoryPage>>(O.none)
+
+  const requestParams = useRef(DEFAULT_REQUEST_PARAMS)
```

Comment:
Why is a reference to `requestParams` needed?

Reply:
as you [can see below](https://github.com/thorchain/asgardex-electron/pull/1249/files#diff-3fc93a20db589613c143bc252a64976566d4c6511c0ace3395293b2fc14a0eccR46) we save previously set params inside of `useObservableState`'s callback to have an opportunity to have small callbacks like [setCurrentPage](https://github.com/thorchain/asgardex-electron/pull/1249/files#diff-3fc93a20db589613c143bc252a64976566d4c6511c0ace3395293b2fc14a0eccR62) which provide **only changed** parameters for actions-request

Reply:
Why don't you use `DEFAULT_REQUEST_PARAMS` all the time to update it with `asset` and / or `page`?

Reply:
mmm. 'cuz they are default 😅 for example:
- user set a filter
- user wants to navigate to another page
- in case using `DEFAULT_REQUEST_PARAMS` filter will be restored to default

Reply:
Ok, I see. Please comment to the code to understand the logic behind the handling of `Partial<LoadActionsParams>` & Co. (which is nice, but not clear at a first view...)